Cargo.toml

```diff
@@ -27,6 +28,7 @@ clap = { version = "4", features = ["color", "derive"] }
 once_cell = "1"
 rand = "0.8"
 tokio = { version = "1", features = ["fs", "time", "rt"] }
+futures = "0.3"
```

Comment:
This doesn't seem to be used?

Reply:
It's for the doctest - I'd imagine most users import `futures`, not `futures_core` 

```
/// use futures::stream::{self, StreamExt};
```

Happy to change if needed!

Reply:
Oh, I'm okay with using `futures_core` for the doctest.

Reply:
Whoops, I think I'd also have to change the doctest, because I wouldn't be able show `futures::stream::iter`:
```rust
# use indicatif::ProgressBar;
use futures::stream::{self, StreamExt};
# async fn test() {
let pb = ProgressBar::new(10);
let stream = pb.wrap_stream(stream::iter(1..=10));
assert_eq!(
    stream.count().await,
    10,
);
# }
```

Reply:
Ah yeah, that's annoying. Okay, can you just add a comment in the `Cargo.toml` for the futures dev-dep why it's there?